### PR TITLE
(Final!) refresh of client

### PR DESCRIPTION
**Pull request by jhowardmsft**
Signed-off-by: John Howard <jhoward@microsoft.com>

Hopefully the last refresh before I kill jhowardmsft/opengcs and transition docker over to Microsoft/opengcs instead later this week. Note it's dependent on 34272 which is the rest of docker moving over to lower-case sirupsen\logrus, imminently about to be merged into master there.

### CODE REVIEW

File: client/config.go [hunk 1]
```diff
@@ -160,8 +160,8 @@ func (config *Config) GenerateDefault(options []string) error {
 	return nil
 }
 
-// validate validates a Config structure for starting a utility VM.
-func (config *Config) validate() error {
+// Validate validates a Config structure for starting a utility VM.
+func (config *Config) Validate() error {
```
Comment (jterry75) [inline client/config.go]: JW why our exporting this now but I see no new callers? Does Docker call this?

**Comment by jterry75** [comment]
You might have to git mv the vendor/github.com/Sirupsen folder to be lower case but not sure

**Comment by jhowardmsft** [comment]
Yeah, that's for the server side of opengcs, not the client. There's no vendoring in the client library - the user of the library (docker being the only one currently) is responsible for vendoring which has already been updated :)

File: client/config.go (hunk 1 above)
Comment (jhowardmsft) [inline client/config.go]: There is in a PR about to be submitted.

**Comment by jterry75** [comment]
Oh. Yuck... I will submit a change for the service side.

**Pull request merged**